pkg/ui/README.md

```diff
@@ -83,6 +83,14 @@ dependencies properly, so a top-level `make build` will rebuild exactly the
 necessary DLLs.) DLLs also make the Webpack configuration rather complicated.
 Still, the tradeoff seems well worth it.
 
+## CCL Build
+
+In CCL builds, code in `pkg/ui/ccl/src` overrides code in `pkg/ui/src` at build
+time, via an import resolution rule. E.g. if a file imports
```

Comment:
Not exactly sure how this works — a webpack DLL? How much of it is in the Makefile vs the webpack configs? A little more detail might be good.

Reply:
webpack: https://github.com/cockroachdb/cockroach/blob/master/pkg/ui/webpack.app.js#L50